┌ Info: 2024-02-21 11:31:12 PgDBController.create_jetelina_id_sequence() error: LibPQ.Errors.DuplicateTable("ERROR:  relation \"jetelina_table_id_sequence\" already exists\n")
└ @ Jetelina.JLog /home/onoke/Jetelina/Jetelina/app/resources/JLog.jl:71
┌ Info: 2024-02-21 11:31:15 SQLAnalyzer.main() start with : 48 hr interval
└ @ Jetelina.JLog /home/onoke/Jetelina/Jetelina/app/resources/JLog.jl:71
┌ Info: 2024-02-22 12:25:56 PgDBController.create_jetelina_id_sequence() error: LibPQ.Errors.DuplicateTable("ERROR:  relation \"jetelina_table_id_sequence\" already exists\n")
└ @ Jetelina.JLog /home/onoke/Jetelina/Jetelina/app/resources/JLog.jl:71
┌ Info: 2024-02-22 12:25:59 SQLAnalyzer.main() start with : 48 hr interval
└ @ Jetelina.JLog /home/onoke/Jetelina/Jetelina/app/resources/JLog.jl:71
┌ Info: 2024-02-24 09:10:29 PgDBController.create_jetelina_id_sequence() error: LibPQ.Errors.DuplicateTable("ERROR:  relation \"jetelina_table_id_sequence\" already exists\n")
└ @ Jetelina.JLog /home/onoke/Jetelina/Jetelina/app/resources/JLog.jl:71
┌ Info: 2024-02-24 09:10:32 SQLAnalyzer.main() start with : 48 hr interval
└ @ Jetelina.JLog /home/onoke/Jetelina/Jetelina/app/resources/JLog.jl:71
┌ Info: 2024-02-24 09:12:02 PgDBController.create_jetelina_id_sequence() error: LibPQ.Errors.DuplicateTable("ERROR:  relation \"jetelina_table_id_sequence\" already exists\n")
└ @ Jetelina.JLog /home/onoke/Jetelina/Jetelina/app/resources/JLog.jl:71
┌ Info: 2024-02-24 09:12:05 SQLAnalyzer.main() start with : 48 hr interval
└ @ Jetelina.JLog /home/onoke/Jetelina/Jetelina/app/resources/JLog.jl:71
┌ Info: 2024-02-24 09:14:22 PgDBController.create_jetelina_id_sequence() error: LibPQ.Errors.DuplicateTable("ERROR:  relation \"jetelina_table_id_sequence\" already exists\n")
└ @ Jetelina.JLog /home/onoke/Jetelina/Jetelina/app/resources/JLog.jl:71
┌ Info: 2024-02-24 09:14:25 SQLAnalyzer.main() start with : 48 hr interval
└ @ Jetelina.JLog /home/onoke/Jetelina/Jetelina/app/resources/JLog.jl:71
┌ Info: 2024-02-24 09:15:54 PgDBController.create_jetelina_id_sequence() error: LibPQ.Errors.DuplicateTable("ERROR:  relation \"jetelina_table_id_sequence\" already exists\n")
└ @ Jetelina.JLog /home/onoke/Jetelina/Jetelina/app/resources/JLog.jl:71
┌ Info: 2024-02-24 09:15:57 SQLAnalyzer.main() start with : 48 hr interval
└ @ Jetelina.JLog /home/onoke/Jetelina/Jetelina/app/resources/JLog.jl:71
┌ Info: 2024-02-24 09:19:03 PgDBController.create_jetelina_id_sequence() error: LibPQ.Errors.DuplicateTable("ERROR:  relation \"jetelina_table_id_sequence\" already exists\n")
└ @ Jetelina.JLog /home/onoke/Jetelina/Jetelina/app/resources/JLog.jl:71
┌ Info: 2024-02-24 09:19:06 SQLAnalyzer.main() start with : 48 hr interval
└ @ Jetelina.JLog /home/onoke/Jetelina/Jetelina/app/resources/JLog.jl:71
┌ Info: 2024-02-24 09:24:07 PgDBController.create_jetelina_id_sequence() error: LibPQ.Errors.DuplicateTable("ERROR:  relation \"jetelina_table_id_sequence\" already exists\n")
└ @ Jetelina.JLog /home/onoke/Jetelina/Jetelina/app/resources/JLog.jl:71
┌ Info: 2024-02-24 09:24:10 SQLAnalyzer.main() start with : 48 hr interval
└ @ Jetelina.JLog /home/onoke/Jetelina/Jetelina/app/resources/JLog.jl:71
┌ Info: 2024-02-24 09:29:31 PgDBController.create_jetelina_id_sequence() error: LibPQ.Errors.DuplicateTable("ERROR:  relation \"jetelina_table_id_sequence\" already exists\n")
└ @ Jetelina.JLog /home/onoke/Jetelina/Jetelina/app/resources/JLog.jl:71
┌ Info: 2024-02-24 09:29:47 SQLAnalyzer.main() start with : 48 hr interval
└ @ Jetelina.JLog /home/onoke/Jetelina/Jetelina/app/resources/JLog.jl:71
┌ Info: 2024-02-24 09:30:36 PgDBController.create_jetelina_id_sequence() error: LibPQ.Errors.DuplicateTable("ERROR:  relation \"jetelina_table_id_sequence\" already exists\n")
└ @ Jetelina.JLog /home/onoke/Jetelina/Jetelina/app/resources/JLog.jl:71
┌ Info: 2024-02-24 09:30:39 SQLAnalyzer.main() start with : 48 hr interval
└ @ Jetelina.JLog /home/onoke/Jetelina/Jetelina/app/resources/JLog.jl:71
┌ Info: 2024-02-24 09:32:10 PgDBController.create_jetelina_id_sequence() error: LibPQ.Errors.DuplicateTable("ERROR:  relation \"jetelina_table_id_sequence\" already exists\n")
└ @ Jetelina.JLog /home/onoke/Jetelina/Jetelina/app/resources/JLog.jl:71
┌ Info: 2024-02-24 09:32:17 SQLAnalyzer.main() start with : 48 hr interval
└ @ Jetelina.JLog /home/onoke/Jetelina/Jetelina/app/resources/JLog.jl:71
┌ Info: 2024-02-24 09:33:21 PgDBController.create_jetelina_id_sequence() error: LibPQ.Errors.DuplicateTable("ERROR:  relation \"jetelina_table_id_sequence\" already exists\n")
└ @ Jetelina.JLog /home/onoke/Jetelina/Jetelina/app/resources/JLog.jl:71
┌ Info: 2024-02-24 09:33:24 SQLAnalyzer.main() start with : 48 hr interval
└ @ Jetelina.JLog /home/onoke/Jetelina/Jetelina/app/resources/JLog.jl:71
┌ Info: 2024-02-24 09:34:11 PgDBController.create_jetelina_id_sequence() error: LibPQ.Errors.DuplicateTable("ERROR:  relation \"jetelina_table_id_sequence\" already exists\n")
└ @ Jetelina.JLog /home/onoke/Jetelina/Jetelina/app/resources/JLog.jl:71
┌ Info: 2024-02-24 09:34:14 SQLAnalyzer.main() start with : 48 hr interval
└ @ Jetelina.JLog /home/onoke/Jetelina/Jetelina/app/resources/JLog.jl:71
┌ Info: 2024-02-24 09:38:36 PgDBController.create_jetelina_id_sequence() error: LibPQ.Errors.DuplicateTable("ERROR:  relation \"jetelina_table_id_sequence\" already exists\n")
└ @ Jetelina.JLog /home/onoke/Jetelina/Jetelina/app/resources/JLog.jl:71
┌ Info: 2024-02-24 09:38:39 SQLAnalyzer.main() start with : 48 hr interval
└ @ Jetelina.JLog /home/onoke/Jetelina/Jetelina/app/resources/JLog.jl:71
┌ Info: 2024-02-24 09:45:55 PgDBController.create_jetelina_id_sequence() error: LibPQ.Errors.DuplicateTable("ERROR:  relation \"jetelina_table_id_sequence\" already exists\n")
└ @ Jetelina.JLog /home/onoke/Jetelina/Jetelina/app/resources/JLog.jl:71
┌ Info: 2024-02-24 09:45:58 SQLAnalyzer.main() start with : 48 hr interval
└ @ Jetelina.JLog /home/onoke/Jetelina/Jetelina/app/resources/JLog.jl:71
┌ Info: 2024-02-24 12:45:35 PgDBController.create_jetelina_id_sequence() error: LibPQ.Errors.DuplicateTable("ERROR:  relation \"jetelina_table_id_sequence\" already exists\n")
└ @ Jetelina.JLog /home/onoke/Jetelina/Jetelina/app/resources/JLog.jl:71
┌ Info: 2024-02-24 12:45:38 SQLAnalyzer.main() start with : 48 hr interval
└ @ Jetelina.JLog /home/onoke/Jetelina/Jetelina/app/resources/JLog.jl:71
┌ Info: 2024-02-26 09:09:09 PgDBController.create_jetelina_id_sequence() error: LibPQ.Errors.DuplicateTable("ERROR:  relation \"jetelina_table_id_sequence\" already exists\n")
└ @ Jetelina.JLog /home/onoke/Jetelina/Jetelina/app/resources/JLog.jl:72
┌ Info: 2024-02-26 09:09:11 SQLAnalyzer.main() start with : 48 hr interval
└ @ Jetelina.JLog /home/onoke/Jetelina/Jetelina/app/resources/JLog.jl:72
┌ Info: 2024-02-26 09:46:33 PgDBController.create_jetelina_id_sequence() error: LibPQ.Errors.DuplicateTable("ERROR:  relation \"jetelina_table_id_sequence\" already exists\n")
└ @ Jetelina.JLog /home/onoke/Jetelina/Jetelina/app/resources/JLog.jl:68
┌ Info: 2024-02-26 09:46:35 SQLAnalyzer.main() start with : 48 hr interval
└ @ Jetelina.JLog /home/onoke/Jetelina/Jetelina/app/resources/JLog.jl:68
